### PR TITLE
ci: add issue reply

### PR DESCRIPTION
**Pull request by xrkffgg**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/qiankun/ISSUE_URL

### CODE REVIEW

File: .github/workflows/issue-reply.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Need Reproduce
+        if: github.event.label.name == 'Need Reproduce'
+        uses: actions-cool/issues-helper@v2.0.0
+        with:
+          actions: 'create-comment'
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. In order to facilitate location and troubleshooting, we need you to provide a realistic example. Please forking these link [codesandbox](https://codesandbox.io/) or clone [qiankun examples](https://github.com/umijs/qiankun/tree/master/examples) to your GitHub repository.
+
+            你好 @${{ github.event.issue.user.login }}, 为了方便定位和排查问题，我们需要你提供一个重现实例，请提供一个尽可能精简的链接 [codesandbox](https://codesandbox.io/) 或也可以直接 clone [qiankun examples](https://github.com/umijs/qiankun/tree/master/examples)，并上传到你的 GitHub 仓库。
```
Comment (xrkffgg) [inline .github/workflows/issue-reply.yml:31]:
```suggestion
            你好 @${{ github.event.issue.user.login }}, 为了方便定位和排查问题，我们需要你提供一个重现实例，请提供一个尽可能精简的链接 [codesandbox](https://codesandbox.io/) 或直接 clone [qiankun examples](https://github.com/umijs/qiankun/tree/master/examples)，并上传到你的 GitHub 仓库。
```

File: .github/workflows/issue-reply.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  reply-helper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: help wanted
+        if: github.event.label.name == 'pr Welcome' || github.event.label.name == 'help wanted'
```
Comment (kuitos) [inline .github/workflows/issue-reply.yml:12]: 现在的 label 应该是 `pr welcome`

**Review by kuitos — CHANGES_REQUESTED** [review]
Great work!

**Comment by kuitos** [comment]
再建一个 pls use issue template 的 replay? 提示用户按模板创建 issue

File: .github/workflows/issue-reply.yml (hunk 2 above)
Comment (xrkffgg) [inline .github/workflows/issue-reply.yml:12]:
```suggestion
        if: github.event.label.name == 'pr welcome' || github.event.label.name == 'help wanted'
```

**Review by kuitos — APPROVED** [review]
Great Job!

**Pull request merged**